ci-operator/step-registry/hypershift/conformance/hypershift-conformance-chain.yaml

```diff
@@ -84,18 +103,19 @@ chain:
       *) echo >&2 "Unsupported cluster type '${CLUSTER_TYPE}'"; exit 1;;
       esac
 
+      set -e
       if [[ -n "${TEST_SKIPS}" ]]; then
           TESTS="$(openshift-tests run --dry-run --provider "${TEST_PROVIDER}" "${TEST_SUITE}")"
           echo "${TESTS}" | grep -v "${TEST_SKIPS}" >/tmp/tests
           echo "Skipping tests:"
           echo "${TESTS}" | grep "${TEST_SKIPS}" || { exit_code=$?; echo 'Error: no tests were found matching the TEST_SKIPS regex:'; echo "$TEST_SKIPS"; return $exit_code; }
           TEST_ARGS="${TEST_ARGS:-} --file /tmp/tests"
       fi
-
       openshift-tests run ${TEST_SUITE} ${TEST_ARGS:-} \
         --provider "${TEST_PROVIDER}" \
         -o "${ARTIFACT_DIR}/e2e.log" \
         --junit-dir "${ARTIFACT_DIR}/junit"
+      exit CSI_CONFORMANCE_EXITCODE
```

Comment:
this is technically accurate, but I think it might be difficult for someone unfamiliar with the context of what the kubevirt platform is doing to understand why we're returning an env var called `CSI_CONFORMANCE_EXITCODE`  at the end of the hypershift conformance.

maybe we can call this something more generic and provide some more context as to why the non zero exit code occurred... Otherwise we're going to have people seeing the conformance tests failed with a non zero exit code, but that all the conformance tests passed with no further context.

maybe something like this would help?

```
if [$? -ne 0]; then
    PLATFORM_FAILURE_CONDITION_MSG="KubeVirt CSI conformance tests contained failures."
fi

.
.
.
if [ -n "$PLATFORM_FAILURE_CONDITION_MSG" ]; then
    echo "Platform [$CLUSTER_TYPE] encountered a failure condition [$PLATFORM_FAILURE_CONDITION_MSG]"
    exit 1
fi
```

maybe there is a better env var than `PLATFORM_FAILURE_CONDITION_MSG` i'm just trying to get us to use something more generic.

Reply:
Yeah I have put a comment in my local version as I realized the same thing. I think your suggestion is better though.